jmeter/scenarios/demo_scenario.jmx

```diff
@@ -70,12 +70,16 @@ without actual network activity. This helps debugging tests.</stringProp>
             <stringProp name="Assertion.test_field">Assertion.response_data</stringProp>
             <boolProp name="Assertion.assume_success">false</boolProp>
             <intProp name="Assertion.test_type">16</intProp>
+            <stringProp name="Assertion.custom_message"></stringProp>
           </ResponseAssertion>
           <hashTree/>
         </hashTree>
       </hashTree>
-      <BeanShellAssertion guiclass="BeanShellAssertionGui" testclass="BeanShellAssertion" testname="BeanShell Assertion" enabled="true">
-        <stringProp name="BeanShellAssertion.query">String response_message = &quot;&quot;;
+      <JSR223Listener guiclass="TestBeanGUI" testclass="JSR223Listener" testname="JSR223 Listener" enabled="true">
+        <stringProp name="cacheKey">true</stringProp>
+        <stringProp name="filename"></stringProp>
+        <stringProp name="parameters"></stringProp>
+        <stringProp name="script">String response_message = &quot;&quot;;
```

Comment:
Did you test this?
Neither JSR223 nor Beanshell listeners has SampleResult object. I'm getting errors, please change SampleResult to sampleResult

Reply:
Fixed